blocks/gallery-block/features/gallery/default.jsx

```diff
@@ -58,30 +72,91 @@ export const GalleryPresentation = ({
+				/* istanbul ignore next */ (current, total) =>
+					phrases.t("global.gallery-page-count-text", { current, total })
+			}
+			enableAutoplay
+			startAutoplayIcon={<Icon name="Play" />}
+			startAutoplayText={phrases.t("global.gallery-autoplay-button")}
+			stopAutoplayIcon={<Icon name="Pause" />}
+			stopAutoplayText={phrases.t("global.gallery-pause-autoplay-button")}
 			autoplayPhraseLabels={{
 				start: phrases.t("global.gallery-autoplay-label-start"),
 				stop: phrases.t("global.gallery-autoplay-label-stop"),
 			}}
-			autoplayPhrase={phrases.t("global.gallery-autoplay-button")}
-			pausePhrase={phrases.t("global.gallery-pause-autoplay-button")}
-			controlsFont={getThemeStyle(arcSite)["primary-font-family"]}
-			pageCountPhrase={
-				/* istanbul ignore next */ (current, total) =>
-					phrases.t("global.gallery-page-count-text", { current, total })
+			enableFullScreen
+			fullScreenShowButton={
+				<button type="button">
+					<Icon name="Fullscreen" />
+					{phrases.t("global.gallery-expand-button")}
+				</button>
+			}
+			fullScreenMinimizeButton={
+				<button type="button">
+					<Icon name="Close" />
+				</button>
+			}
+			adElement={/* istanbul ignore next */ <AdBlock />}
+			adInterstitialClicks={interstitialClicks}
+			nextButton={
+				<Carousel.Button id={id} label="Next Slide">
+					<Icon className={`${BLOCK_CLASS_NAME}__track-icon`} fill="white" name="ChevronRight" />
+				</Carousel.Button>
+			}
+			previousButton={
+				<Carousel.Button id={id} label="Previous Slide">
+					<Icon className={`${BLOCK_CLASS_NAME}__track-icon`} name="ChevronLeft" />
+				</Carousel.Button>
 			}
-			adElement={/* istanbul ignore next */ () => <AdBlock />}
-			interstitialClicks={interstitialClicks}
-			displayTitle={!hideTitle}
-			displayCaption={!hideCaption}
-			displayCredits={!hideCredits}
-		/>
+		>
+			{contentElements.map((galleryItem, itemIndex) => (
+				<Carousel.Item
+					label={phrases.t("global.gallery-page-count-text", { itemIndex, galleryLength })}
+					key={`gallery-item-${galleryItem.url}`}
+				>
+					<MediaItem
+						caption={!hideCaption ? galleryItem.caption : null}
+						credit={
+							!hideCredits ? formatCredits(galleryItem.vanityCredits || galleryItem.credits) : null
+						}
+						title={!hideTitle ? galleryItem.subtitle : null}
+					>
+						<div className={`${BLOCK_CLASS_NAME}__image-wrapper`}>
+							<Image
+								src={imageANSToImageSrc(galleryItem)}
+								resizerURL={resizerURL}
+								resizedOptions={{ auth: galleryItem.auth[resizerAppVersion] }}
+								// 16:9 aspect ratio
+								width={800} // matches width of previous gallery
+								height={450}
+								alt={galleryItem.alt_text}
+								resizerAppVersion={resizerAppVersion}
+								responsiveImages={[150, 375, 500, 800, 1500, 2000]}
+								sizes={[
+									{
+										isDefault: true,
+										// 75vh max height
+										// if 16/9 aspect ratio, then roughly 40vw expected
+										sourceSizeValue: "40vw",
```

Comment:
Oof, I don't have a solve for this, but that value will be different on mobile because of the flipped aspect ratio.

Reply:
480px:
![image](https://user-images.githubusercontent.com/2287238/187774059-3b4b0cb0-314e-43cd-86e0-f79b61e35aeb.png)
 :/

Reply:
I see you changed the width on this... double checking

Reply:
ya it was looking weirdly without it. I think 100vw might be ok if we constrain max height

Reply:
Actually, since you only have one size, there's no point in including the sizes array,  Removing it changes nothing as far as I can tell here.
![image](https://user-images.githubusercontent.com/2287238/187776760-bcfe0e52-7c42-40a0-b955-afa65a27c081.png)

Reply:
I think it would be useful if you had a media query size:  https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/sizes#examples

Reply:
I assume removing it would be that 100vw you were referring to

Reply:
not going to sweat this... from what I understand of it, it might be a good plan to pass it the size of the container it's in so it can properly determine the image to use

Reply:
It's removed